### DISCUSSION TITLE
Unreal Engine 5.5

### DISCUSSION THREAD

**Discussion by R0NIN-6** [ErikHedberg/Terminal-Ballistics-Example-Project#15] [General]
Hi is there an ETA on when the plugin will be updated to support UE version 5.5?

**Comment by ErikHedberg** [comment]
Currently awaiting approval from Epic. The update (v1.0.72) was submitted a couple weeks ago that this point.

**Comment by AlphaChip** [comment]
any update on this? Three weeks now seems like an abnormally long amount of time.

**Reply by wupy36** [reply]
FAB has been taking a really long time to update plugins especially now with 5.5 being released. 
I had a update pending since Oct 30th and it went through yesterday finally.
